src/main/paradox/docs/api/current/kg/views/sparql-view-api.md

```diff
@@ -104,7 +105,7 @@ Request
 Response
 :   @@snip [view-fetched.json](../../assets/views/view-statistics.json)
 
-where:
+where...
```

Comment:
I think `:` makes more sense here than `...`.

Reply:
Yeah, we can change then everywhere, cause on the other places we have 3 dots